Sources/MoyaSugar/MoyaSugarProvider.swift

```diff
@@ -9,6 +11,7 @@ open class MoyaSugarProvider<Target: SugarTargetType>: MoyaProvider<Target> {
     endpointClosure: @escaping EndpointClosure = MoyaProvider.defaultEndpointMapping,
     requestClosure: @escaping RequestClosure = MoyaProvider.defaultRequestMapping,
     stubClosure: @escaping StubClosure = MoyaProvider.neverStub,
+    callbackQueue: DispatchQueue? = nil,
```

Comment:
I think we have to pass this parameter to `super.init()` either. And we may need some test of it.

Reply:
I thought it would respect to `super.init(...)`. I'll change it.

Reply:
Oops!